.github/CODEOWNERS

```diff
@@ -1,8 +1,9 @@
-.github/**/*.yml    @centreon/owners-pipelines
-
-*.po                @centreon/owners-doc
-*.md                @centreon/owners-doc
-*.mdx               @centreon/owners-doc
+.github/**          @centreon/owners-pipelines
 
```

Comment:
```suggestion

*.po @centreon/owners-doc

```

Reply:
we still need to be automatically added to PRs that contain .po files

Reply:
there's no po files in this repository !!!

Reply:
indeed. my bad.